subsys/usb/class/cdc_acm.c

```diff
@@ -1075,9 +1042,11 @@ static const struct uart_driver_api cdc_acm_driver_api = {
 #endif /* CONFIG_USB_COMPOSITE_DEVICE */
 
 #define DEFINE_CDC_ACM_DEV_DATA(x)					\
+	RING_BUF_DECLARE(rx_ringbuf_##x, 512);				\
```

Comment:
Is it an option to make the ring buffer size configurable in Kconfig? I tested the patch and ran into issues when the buffer was filled and started dropping data before the callback was served from the system workqueue.

Reply:
I can make it configurable.

Reply:
Thanks!

Reply:
Done